pacsim:
  timeouts:
    start: 60 # time to trigger first timekeeping gate
    acceleration: 25
    autocross: 300
    skidpad: 90
    trackdrive_first: 300
    trackdrive_total: 1000
  oc_detect: true
  doo_detect: true
  uss_detect: true
  finish_validate: true
